BoxContentSDK/BoxContentSDK/Clients/BOXContentClient+File.h

```diff
@@ -362,6 +362,21 @@
                                                                   toLocalFilePath:(NSString *)localFilePath
                                                                    representation:(BOXRepresentation *)representation;
 
+/**
+ *  Generate a request to download a given representation of a file to a local filepath.
+ *
+ *  @param fileID          File ID.
+ *  @param localFilePath   Local filepath.
+ *  @param representation  BOXRepresentation to be downloaded
+ *  @param associateID A unique ID used to recover background tasks. Providing an ID associated with a running background
```

Comment:
@param associateID   A unique ID to associate with a background download task. Required together with localFilePath to trigger a background download.